README.md

```diff
@@ -4,13 +4,36 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1) Asymptotic analysis normally only accounts for the average or worst case scenarios in algorithms, but in the real world these cases are often different and unpredictable.
+     
+  2) In the real world, the efficiency of these algorithms can be altered by uncontrollable factors such as the hardware or programming language the algorithm is being ran on.
```

Comment:
Both of those are very controllable.

Reply:
I meant more that the algorithms do not have control over these factors, and so their performance may only be optimized for the programming language or hardware it was written for.

Reply:
...so what you're saying is that the experiments were run in different environments?